### PR TITLE
Updating README of the example DataOps repo

### PR DESCRIPTION
**Pull request by dse0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+```
+
+To create a Preview Build:
+```
+$ glean preview (subdirectory)
+# Example:
+$ glean preview ./01_subscription
+```
+
+*Note*: These examples will only successfully build if you have the corresponding demo dataset configured in your Glean project.
```
Comment (trucklos) [inline README.md]: I think these examples will just build on a demo project right?  So maybe just point them to that.

"You will need a bigquery connection called `whatever-the-name-is` with the corresponding datasets.  You can get these by adding a demo sandbox project (with link to docs on how to do that)."

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
-# exmaples
-Demo and examples to build in Glean
+# Glean project examples
+
```
Comment (trucklos) [inline README.md:2]: maybe point people at demo.glean.io?  I actually think most of the content from the landing page highlighting usecases could be good here too.  Can do that on another pass too

**Review by trucklos — APPROVED** [review]
lgtm!

File: README.md (hunk 1 above)
Comment (dse0) [reply README.md]: Ah right, cool

**Pull request merged**